lab3/middlebox.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+"""
+David to implement
+"""
+
+from switchyard.lib.address import *
+from switchyard.lib.packet import *
+from switchyard.lib.packet.util import *
+from switchyard.lib.userlib import *
+from threading import *
```

Comment:
I think this `threading import` was provided in the course provided starting point, but it doesn't appear that we use anything from it. So this should be removed.